src/components/ChatMessage.tsx

```diff
@@ -0,0 +1,35 @@
+import {
+    IonCard,
+    IonCardContent,
+    IonCardHeader,
+    IonImg,
+    IonCardTitle,
+} from "@ionic/react";
+import "./ChatMessage.scss";
+
+interface ChatMessageProps {
+    sent?: boolean,
+    src?: string,
+    text: string,
+    username?: string
```

Comment:
Could we add date and time sent as well and display it next to the username?

Reply:
Oh, yes! Maybe at the bottom of the message in small letters, or to the right/left side?